### ISSUE TITLE
Setup

### ISSUE THREAD

**Issue by tim4849** [croedel/M-TEC-API#1]
I downloaded the files from github to my raspi, but I can't get them to work, everything is configured as in the readme:
`<html><body>
<pre class="markdown-highlight"><code class="hljs ruby" data-lines="1">

pi@iob1:~/MTEC-API $ python3 MTEC_client.py
--

</code></pre><pre class="markdown-highlight"><code class="hljs ruby" data-lines="1">                            <div class="hljs-ln-line"><span class="hljs-symbol">ModuleNotFoundError:</span> No <span class="hljs-class"><span class="hljs-keyword">module</span> <span class="hljs-title">named</span> '<span class="hljs-title">yaml</span>'</span></div></code></pre>
</body>
</html>
`
What am I doing wrong?

Many greetings M

**Comment by hschuetz1208** [comment]
Have you installed the pyyaml package?

`pip install pyyaml`

**Comment by croedel** [comment]
Yes, the command @hschuetz1208 proposed should solve the issue. I also added this to the Readme

**Comment by proj8134** [comment]
Hi 
I've installed the Python scripts and configured the login data. The MTEC_client.py work well through menue 1 & 2.
On menue 3 I get this error. 
Please select station #: 3
Invalid #
[ERROR] MTECapi.py: Error while retrieving device list for stationId 'None': {'msg': 'Lack parameter', 'code': '3010130', 'time': '2024-03-11T12:53:56.072+00:00', 'data': None}
Any idea what I do wrong?

**Comment by proj8134** [comment]
If I would read carefully the output....

Please select: 3
  (#0) StationId '1234560' : Test - Town - 12346 / ABC
Please select station : 0
  (#0) DeviceId '987456' : Energy Butler 8kW
Please select device #: 0
